### PR TITLE
Fix bug in Explorer - promise is undefineds: Add es6-promise

### PR DESCRIPTION
**Pull request by noanu**

### CODE REVIEW

File: EmbedSample/EmbedSample.csproj [hunk 1]
```diff
@@ -166,6 +166,7 @@
     <Content Include="js\list.js" />
     <Content Include="js\modal.js" />
     <Content Include="js\theming.js" />
+    <Content Include="lib\js\es6-promise.auto.min.js" />
```
Comment (ranbreuer) [inline EmbedSample/EmbedSample.csproj]: maybe sort and join with other lib/*?
also, where are the other js lib files?

File: ThirdPartyNotices.txt [hunk 2]
```diff
@@ -35,3 +35,27 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
```
Comment (ranbreuer) [inline ThirdPartyNotices.txt:38]: do we need to put something on the js file? similar to other files?

**Review by ranbreuer — APPROVED** [review]

**Review by ali-hamud — APPROVED** [review]

File: EmbedSample/EmbedSample.csproj (hunk 1 above)
Comment (noanu) [reply EmbedSample/EmbedSample.csproj]: thanks - I sorted it.

File: ThirdPartyNotices.txt (hunk 2 above)
Comment (noanu) [reply ThirdPartyNotices.txt:38]: I put. Didn't you see?

Comment (ranbreuer) [reply ThirdPartyNotices.txt:38]: saw now

**Pull request merged**